pr-deploy/app.yaml

```diff
@@ -0,0 +1,2 @@
+runtime: nodejs
+env: flexible
```

Comment:
I've seen this setting in other files. Needed here?
```yaml
beta_settings:
  cloud_sql_instances: amp-pr-deploy-bot:us-central1:amp-pr-deploy-bot
```

Reply:
Nope, pr deploy doesn't use Cloud SQL